docker-compose.yaml

```diff
@@ -15,7 +15,11 @@ services:
     # Add the "stop_signal: SIGINT" option to use SIGINT signal for stopping the container gracefully
     stop_signal: SIGINT
     environment:
-      MODEL_PATH: /models/wizardLM-7B-HF
+      MODEL_PATH: models/vicuna-AlekseyKorshuk-7B-GPTQ-4bit-128g
```

Comment:
I'll approve the PR but I think we need a different way to define these. One which does not require thinking when pulling/pushing commits when different Devs use different models.